docs/getting-started/cli.md

```diff
@@ -0,0 +1,200 @@
+---
+slug: cli
+title: CLI-only workflow 
+hide_title: true
+description: Learn how to set up Semgrep, scan your first project for security issues, and view your findings in the CLI.
+tags:
+  - quickstart
+  - cli
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import MoreHelp from "/src/components/MoreHelp";
+import Install from "/src/components/procedure/_install-cli.mdx";
+import Login from "/src/components/procedure/login-activate.mdx";
+
+# CLI-only workflow
+
+Learn how to set up Semgrep, scan your first project for security issues, and view your findings in the CLI.
+
+## Prerequisites
+
+See [Prerequisites](/prerequisites/) to ensure that your machine meets Semgrep's requirements.
+
+## Set up Semgrep
+
+<Install />
+
+## Log into your Semgrep account
+
+<Login />
+
+## Scan your project
+
+<Tabs
+    defaultValue="pro"
+    values={[
+    {label: 'Semgrep Pro', value: 'pro'},
+    {label: 'Semgrep OSS', value: 'oss'},
+    ]}
+>
+
+<TabItem value='pro'>
+
+Navigate to the root of your repository, and run your first scan. The specific command you use depends on how you want to view the results.
+
+To view the results in the CLI:
+
+```console
+semgrep ci
+```
+
+To export the results to a SARIF file:
+
+```console
+semgrep ci --sarif --output=semgrep.sarif
+```
+
+To export the results to a JSON file:
+
+```console
+semgrep ci --json --output=semgrep.json
+```
+
+### Scan your project with a specific ruleset
```

Comment:
We actually do not want people to use `semgrep ci --config auto`:

https://semgrepinc.slack.com/archives/C8HRFBPSQ/p1699643585940479